### ISSUE TITLE
onMoveNode

### ISSUE THREAD

**Issue by LeahPike** [500tech/angular2-tree-component#71]
Version **1.3.2**
I can drag and drop my nodes beautifully, it looks great... but the onMoveNode event isn't firing?

```
<Tree [nodes]="nodes" [options]="treeOptions" (onMoveNode)="onMoveNode($event)"></Tree>
```

```
treeOptions = {
    allowDrag: true
}
```

```
onMoveNode($event) {
  console.log(
    "Moved",
    $event.node.data.name,
    "to",
    $event.to.node.data.name,
    "at index",
    $event.to.index);
}
```

I have just done the "fix" mentioned in #68, might be related?

**Comment by adamkleingit** [comment]
Nope, this is a bug - forgot to add onMoveNode as Output.
Will be fixed in next version.

**Comment by adamkleingit** [comment]
Fixed in 1.3.3
